biggraphite/drivers/memory.py

```diff
@@ -45,7 +44,6 @@ def __init__(self):
             sortedcontainers.SortedDict)
         self._name_to_metric = {}
         self._directory_names = sortedcontainers.SortedSet()
-        self.__downsampler = _downsampling.Downsampler()
```

Comment:
do not remove it, this is explicitely added to memory and cassandra because at some point some drivers won't have the use for a "soft" downsampler and will be able to do that in the database directly.